### PR TITLE
Vodafone Station integration doc

### PR DESCRIPTION
**Pull request by chemelli74**
## Proposed change

Introduce documentation for Vodafone Station integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/94032
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4515
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/vodafone_station.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - '@paoloantinori'
+  - '@chemelli74'
+ha_iot_class: Local Polling
+ha_platforms:
+  - device_tracker
+ha_ssdp: false
+ha_integration_type: integration
+---
+
+The Vodafone Station integration allows you to control your [Vodafone Station](https://www.vodafone.it/privati/area-supporto/assistenza-dispositivi/vodafone-station.html) based router.
```
Comment (mib1185) [inline source/_integrations/vodafone_station.markdown:19]: is there an English page to be linked available?

Comment (chemelli74) [reply source/_integrations/vodafone_station.markdown:19]: Unfortunately not.

File: source/_integrations/vodafone_station.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: Vodafone Station
+description: Instructions on how to integrate Vodafone Station routers into Home Assistant.
+ha_category:
+  - Presence Detection
+ha_release: '2023.08'
```
Comment (bieniu) [inline source/_integrations/vodafone_station.markdown]:
```suggestion
ha_release: 2023.9
```

Comment (chemelli74) [reply source/_integrations/vodafone_station.markdown]: Eh indeed, time passes ;-)

**Review by bieniu — APPROVED** [review]
Looks good 👍

**Comment by chemelli74** [comment]
> Looks good 👍

Thx !

**Pull request merged**